### PR TITLE
Issue-238: Adding silent, teaching, collaboration hangout tooltips on…

### PR DESCRIPTION
**Pull request by abhiii5459**
… buttons

### CODE REVIEW

File: client/templates/status/update-status.html [hunk 1]
```diff
@@ -4,10 +4,10 @@
     </div>
     <div class="status-actions">
         {{#if isWorking type}}
-        <span class="interest">I'm interested in:</span>
-        <button class="btn btn-circle btn-danger btn-outline btn-hangout-status" data-type="silent"><i class="fa fa-microphone-slash"></i></button>
-        <button class="btn btn-circle btn-warning btn-outline btn-hangout-status" data-type="teaching"><i class="fa fa-user"></i></button>
-        <button class="btn btn-circle btn-success btn-outline btn-hangout-status" data-type="collaboration"><i class="fa fa-users"></i></button>
+        <span class="interest">I'm interested in joining a :</span>
+    <button class="btn btn-circle btn-danger btn-outline btn-hangout-status" data-type="silent" title="Silent Hangout"><i class="fa fa-microphone-slash"></i></button>
```
Comment (camilleriluke) [inline client/templates/status/update-status.html]: Uneven character spacing

**Comment by abhiii5459** [comment]
Fix for #238 right here.

**Comment by camilleriluke** [comment]
Apart from the spacing, LGTM

File: client/templates/status/update-status.html (hunk 1 above)
Comment (abhiii5459) [inline client/templates/status/update-status.html]: Fixing!

**Comment by abhiii5459** [comment]
Fixed indenting post review for fix #238

**Comment by camilleriluke** [comment]
Nice one on your first PR 🚀

**Comment by lpatmo** [comment]
Looks great -- thanks for writing and solving the issue, @abhiii5459, and thanks @camilleriluke for reviewing it! There's another longer open issue that another contributor is taking. 

Merged into `master`; the changes will go out to the live site in the next deployment to production.

**Comment by abhiii5459** [comment]
Thanks @lpatmo . On to the next one 👍

**Pull request merged**